routers/repo/reviews.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package repo
+
+import (
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/google/git-appraise/repository"
+	"github.com/google/git-appraise/review"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/base"
+	"code.gitea.io/gitea/modules/context"
+)
+
+const (
+	tplReviews base.TplName = "repo/review/list"
+)
+
+// MustAllowReviews check if repository enable reviews
+func MustAllowReviews(ctx *context.Context) {
+	if !ctx.Repo.Repository.AllowsReviews() {
+		ctx.Handle(404, "MustAllowReviews", nil)
+		return
+	}
+}
+
+// Review struct to Issue representation
+type Review struct {
```

Comment:
I tried but I need that index field will be string, because appraise doesn't have unique in index